addons/mail/static/src/new/thread/message.js

```diff
@@ -159,6 +160,26 @@ export class Message extends Component {
         return this.messaging.state.threads[this.props.threadLocalId].id === "inbox";
     }
 
+    /**
+     * @returns {boolean}
+     */
+    get shouldDisplayAuthorName() {
+        if (!this.props.isInChatWindow) {
```

Comment:
Why use a props here ? You have a `this.env.inChatWindow` that should give you this information.

Reply:
I didn't know about that, looks great 👍️